docs/api/util.rst

```diff
@@ -45,11 +45,19 @@ Async
 Aliases
 ~~~~~~~
 
-These functions provide simple aliases for those implemented in :any:`asyncio`, with
-fallbacks for older versions of Python.
+Falcon used to provide aliases for the below functions implemented in
+:mod:`asyncio`, with fallbacks for older versions of Python:
 
-.. autofunction:: falcon.get_running_loop
-.. autofunction:: falcon.create_task
+* ``falcon.get_running_loop()`` → :func:`asyncio.get_running_loop`
+
+* ``falcon.create_task(coro, *, name=None)``  → :func:`asyncio.create_task`
+
+However, as of Falcon 4.0+, these aliases are identical to their :mod:`asyncio`
+counterparts on all supported Python versions. (They are only kept for
+compatibility purposes.)
+
+Simply use :func:`asyncio.get_running_loop` & :func:`asyncio.create_task`
```

Comment:
```suggestion
Simply use :func:`asyncio.get_running_loop` or :func:`asyncio.create_task`
```

Reply:
Hm, I meant _and_. But _or_ does the job too if it feels more natural to you.

Reply:
i did not like the & there. and worked too. overall all 3 are likely fine